monitor/api/v10.py

```diff
@@ -13,7 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from flask import request
```

Comment:
This is a unused import? If yes, please comment this modification in the description of this PR.

Reply:
Done!